dashboard/pre_processing_data.py

```diff
@@ -11,7 +11,7 @@
 
 from gettsim import set_up_policy_environment
 from gettsim.piecewise_functions import piecewise_polynomial
-from gettsim.taxes.eink_st import st_tarif
+from gettsim.taxes.eink_st import _st_tarif
```

Comment:
If renaming anyhow, why not make it 

```suggestion
from gettsim.taxes.eink_st import _eink_st_tarif
```
?